app/views/course/assessment/question/programming/_question.json.jbuilder

```diff
@@ -14,6 +14,7 @@ json.question do
     @programming_question.attachment.present? : @assessment.autograded?
   json.has_auto_gradings @programming_question.auto_gradable? &&
     @programming_question.answers.without_attempting_state.count > 0
+  json.has_submissions @programming_question.answers.without_attempting_state.count > 0
```

Comment:
Assign `@programming_question.answers.without_attempting_state.count` to a variable first, otherwise this line and the line before will generate 2 queries.